admin_guide/revhistory_admin_guide.adoc

```diff
@@ -8,6 +8,24 @@
 
 // do-release: revhist-tables
 
+== Fri Feb 23 2018
+
+// tag::admin_guide_fri_feb_23_2018[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Fri Feb 23 2018
+n|xref:../admin_guide/scheduler.adoc#admin-guide-scheduler[Default Scheduling]
```

Comment:
@mburke5678 This path should be /admin_guide/scheduling/scheduler.adoc#scheduler-policy

I am adding a revision history to 3.6 now and can fix it there, but I would check these paths if you're adding to other product branches. I recommend running `asciibinder build` to check links each time just to be sure. I caught several of my own mistakes this morning doing the same.